pyzebra/comm_export.py

```diff
@@ -0,0 +1,43 @@
+path = 'C:\\Users\\Jakub\\Desktop\\'
+name = 'test'
+extension = '.comm'
+
+
+def fill_string(string, total_lenght):
+    """fills string to tatal_length with whitespaces
+    :arg string - string to fill
+    :arg total_lenght - length of the final string
+    :return string of the lenght of total_lenght
+    """
+    white_spaces = ' '*total_lenght
+    return_string = white_spaces[:-len(str(string))] + str(string)
+    return return_string
+
+
+def export_comm(data, name):
+    """exports data in the *.comm format
+    :arg data - data to export, is dict after peak fitting
+    arg: name - name of the exported file
+    """
+    out_file = open(str(path+name+extension), "w")

```

Comment:
Here it's important to use context manager (the 'with' statement), otherwise, in case of exception the file won't be closed.
Something like that will look better:
```
with open(str(path+name+extension), "w") as out_file:
    for keys in data["Measurements"]:
    ....
```
The line `out_file.close()` is not needed.

Reply:
done :)